phoenix-core/src/main/java/org/apache/phoenix/mapreduce/index/IndexTool.java

```diff
@@ -573,6 +610,21 @@ public Job getJob() {
         return job;
     }
 
+    private void createIndexToolOutputTable(Connection connection) throws Exception {
+        ConnectionQueryServices queryServices = connection.unwrap(PhoenixConnection.class).getQueryServices();
+        Admin admin = queryServices.getAdmin();
+        if (admin.tableExists(TableName.valueOf(OUTPUT_TABLE_NAME))) {
+            return;
+        }
+        HTableDescriptor tableDescriptor = new
+                HTableDescriptor(TableName.valueOf(OUTPUT_TABLE_NAME));
+        tableDescriptor.setValue("DISABLE_TABLE_SOR", "true");
```

Comment:
Not a property that open source HBase or Phoenix recognizes.

Reply:
I will remove it in the open source versions.